### ISSUE TITLE
mouse cursor is erratic

### ISSUE THREAD

**Issue by markwkidd** [libretro/hatari#31]
There is a bug listed at the bottom of the README that doesn't have a corresponding github issue yet.

```HATARI GUI is not functional because of a mouse bug (too much speed). Usually you don't need to enter the GUI. It was fixed in the P-UAE core with the same GUI, so a patch will be done for this core.```

My guess is that the patch being referred to in P-UAE is this commit: https://github.com/libretro/libretro-uae/commit/10e5c3ca71b5db26e619012b3a8463fc9e6a6498

@grant2258 this is what I was mentioning on discord

**Comment by markwkidd** [comment]
There may also be some ideas for input in this recent fork of libretro P-UAE by @qclart: https://github.com/qclart/libretro-uae/commits/master

**Comment by DamsRoxxx** [comment]
@markwkidd No, this was fixed before my PR 34. I only fixed the windows build.

I think the fix in P-UAE could be ported to Hatari as the port are very similar. Never had time to look at it.
